lib/elixir/lib/module.ex

```diff
@@ -830,6 +830,9 @@ defmodule Module do
         false ->
           {name, arity} = tuple
           raise "cannot make function #{name}/#{arity} overridable because it was not defined"
+        {{{:def, {name, arity}}, :defmacrop, _line, _file, _check, _location, _defaults}, _clauses} ->
+          raise "cannot make private macro #{name}/#{arity} overridable because overriding " <>
```

Comment:
Yes this error message is not the best, I just tries to make it similar to the one above. Comma sounds good though.